libs/networking/sources/Sync.cpp

```diff
@@ -248,7 +258,12 @@ void net::Sync::operator()()
         Snapshot &previous = find_last_ack(it - clients.begin());
         Snapshot current(_registry.getTick(), _registry);
 
-        auto packet = constructUpdatePacket(_playerHash, previous, current);
+        auto packet = constructUpdatePacket(
+            previous, current,
+            [this](engine::Entity entity, uint8_t component_id) {
+                return this->canSend(this->_playerNumber, entity, component_id);
```

Comment:
En vrai pourquoi pas mettre player number en protected (ou faire un getter)

Reply:
ça change quoi ?

Reply:
Bah pas besoin de le passer en paramètre en plus alors que c’est déjà inclus dans this ?

Reply:
ah pas con, je vais le mettre en protected